exporter/otlpexporter/README.md

```diff
@@ -50,3 +53,8 @@ Several helper files are leveraged to provide additional capabilities automatica
 - [gRPC settings](https://github.com/open-telemetry/opentelemetry-collector/blob/main/config/configgrpc/README.md)
 - [TLS and mTLS settings](https://github.com/open-telemetry/opentelemetry-collector/blob/main/config/configtls/README.md)
 - [Queuing, retry and timeout settings](https://github.com/open-telemetry/opentelemetry-collector/blob/main/exporter/exporterhelper/README.md)
+
+[beta]: https://github.com/open-telemetry/opentelemetry-collector-contrib#beta
+[contrib]: https://github.com/open-telemetry/opentelemetry-collector-releases/tree/main/distributions/otelcol-contrib
+[core]: https://github.com/open-telemetry/opentelemetry-collector-releases/tree/main/distributions/otelcol
+[stable]: https://github.com/open-telemetry/opentelemetry-collector-contrib#stable
```

Comment:
I would consider to move that document here, since usually we depend in contrib on core not the other way around, separate PR probably.